app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,12 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
Elf and Dwarf should also inherit from ABC -- it will show that you don't want to have an ability to create an instance from the class

Reply:
Done